doc/releases/release-notes-3.4.rst

```diff
@@ -741,6 +741,20 @@ Libraries / Subsystems
     fs_mgmt, :kconfig:option:`CONFIG_FLASH` and
     :kconfig:option:`CONFIG_IMG_MANAGER` are needed to enable MCUmgr img_mgmt.
 
+* POSIX API
+
+  * Improved the locking strategy for eventfd_read() and eventfd_write(). This
```

Comment:
please use <code>:c:func:\`foobar()\`<code>

Reply:
Does that work even when there is no ReST or Doxygen documentation for `foo()`?

Reply:
mmm right, maybe just go with simple backticks then, if you don't mind while you fix the other typo. Thank you!
FWIW :c:func seems to work in this case, but since it's no different than using simple backticks...

Reply:
Fixed!